datashader/geo.py

```diff
@@ -5,19 +5,30 @@
 
 from __future__ import division, absolute_import
 
+import warnings
+
 import pandas as pd
 import numpy as np
 import datashader.transfer_functions as tf
 
 from datashader import Canvas
 from datashader.colors import rgb
-from datashader.utils import ngjit, lnglat_to_meters   # noqa (API import)
+from datashader.utils import (
+    VisibleDeprecationWarning, ngjit, lnglat_to_meters   # noqa (API import)
+)
 from xarray import DataArray
 
 __all__ = ['mean', 'binary', 'slope', 'aspect', 'ndvi', 'hillshade', 'generate_terrain',
            'lnglat_to_meters']
 
 
+warnings.warn(
+    "The datashader.geo module is deprecated as of version 0.11.0. "
+    "Its contents have migrated to the xarray_spatial library.",
+    VisibleDeprecationWarning
```

Comment:
Should we link to the websites for xarray-spatial and spatialpandas libraries?  It's odd for an error message to do that in general, but those aren't necessarily easy to find. A google search for "xarray-spatial" doesn't show the right library on the first page of results.